CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Delete `error` label on Checkbox and RadioButton.
```

Comment:
Ce changement ne concerne que le styleguide ? Car actuellement tu ne modifie pas directement mes composants `Checkbox` et `RadioButton`. L'intitulé porte à confusion.
Peut être que qqch dans le genre 
`Fix: Delete `error` label on Checkbox and RadioButton guideline example.` serait plus approprié ?